tests/dbal/migration/if_params.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+class phpbb_dbal_migration_if_params extends \phpbb\db\migration\migration
+{
+	function update_schema()
+	{
+		return array();
+	}
+
+	function update_data()
+	{
+		return array(
+			array('if', array(
+				true,
+				array('custom', array(array(&$this, 'test'), array('true'))),
```

Comment:
Not that the documentation use the same branches than phpbb/phpbb (3.1.x, 3.2.x and master)